service/matching/config.go

```diff
@@ -46,7 +46,7 @@ type (
 		RangeSize                    int64
 		GetTasksBatchSize            dynamicconfig.IntPropertyFnWithTaskQueueInfoFilters
 		UpdateAckInterval            dynamicconfig.DurationPropertyFnWithTaskQueueInfoFilters
-		IdleTaskqueueCheckInterval   dynamicconfig.DurationPropertyFnWithTaskQueueInfoFilters
+		IdleTaskqueueCheckInterval   dynamicconfig.DurationPropertyFn
```

Comment:
this dynamic config should still be valid, i.e. operator should have the ability to control each individual task queue just in case

Reply:
Pretty strong disagree on this. I noted our previous conversation with Max as well.

Each queue can still get its own liveness threshold, this config applies to how frequently we _check_ the liveness. I submit that using a global configuration for the check interval is fine and of course dramatically simplifies GC scheduling.

Reply:
>Each queue can still get its own liveness threshold, this config applies to how frequently we check the liveness.

i see